Tools/module_config/output_groups_from_timer_config.py

```diff
@@ -80,6 +80,8 @@ def get_timer_groups(timer_config_file, verbose=False):
             max_num_channels = 16 # Just add a fixed number of timers
             timers = [str(i) for i in range(max_num_channels)]
             dshot_support = {str(i): False for i in range(max_num_channels)}
+            for i in range(8): # First 8 channels support dshot
```

Comment:
But not on all HW? I think this may create false Dshot support on other HW.

On the STM32 It has to have the DMA set, if not it is not Dshot capable.

Reply:
I missed this is in the imxrt only section.